test/plugins/tokens.test.js

```diff
@@ -16,7 +16,7 @@ const getTokenMock = (token) => {
 
     mock.update = sinon.stub();
     mock.regenerate = sinon.stub();
-    mock.toJson = sinon.stub().returns(token);
+    mock.toJson = sinon.stub().returns(hoek.clone(token));
```

Comment:
When did we introduce `hoek`? For cloning and combining object maps, ES6 has [`Object.assign`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign)